_sass/moving/_layout.scss

```diff
@@ -308,34 +308,39 @@
  */
 .about-page {
   display: inline-block;
-}
 
-.about-your-name {
-  display: flex;
-  flex-direction: row;
-  margin-top: 1.0rem;
-  @include relative-font-size(1.5);
-}
+  .header {
```

Comment:
Instead of header, I think ‘about-header’ will be better.

Reply:
fix